microsoftgraph/microsoft-graph-explorer-v4
# Task: Export selected resources to postman collection

## Overview

Fixes #1110 
Fixes #1111 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Go to https://jolly-sand-0ac78c710-1277.centralus.azurestaticapps.net and navigate to resources
* On the resource that you pick, select the more actions button and choose add to collection
* Notice a preview  button is now visible and active
* Click on the preview button to expose a panel with the paths
* Click on the download as postman collection button and view the collection in your downloads folder
